contrib/resources/translations/en.lng

```diff
@@ -542,11 +542,25 @@ Select the frame presentation mode:
   cfr:   Always present DOS frames at a constant frame rate.
   vfr:   Always present changed DOS frames at a variable frame rate.
 .
-:CONFIG_OUTPUT
+:CONFIGITEM_OUTPUT_OPENGL_DEFAULT
 Video system to use for output ('opengl' by default).
-'texture' and 'opengl' use bilinear interpolation, 'texturenb' and
-'openglnb' use nearest-neighbour (no-bilinear). Some shaders require
-bilinear interpolation, making that the safest choice.
+Some shaders require bilinear interpolation, making that the safest choice.
+.
+:CONFIGITEM_OUTPUT_TEXTURE_DEFAULT
+Video system to use for output ('texture' by default).
+Some shaders require bilinear interpolation, making that the safest choice.
+.
+:CONFIGITEM_OUTPUT_OPENGL
+  opengl:     Uses bilinear interpolation.
+.
+:CONFIGITEM_OUTPUT_TEXTURE
+  texture:    Uses bilinear interpolation.
+.
+:CONFIGITEM_OUTPUT_OPENGLNB
+  openglnb:   Uses nearest-neighbour interpolation (no-bilinear).
+.
+:CONFIGITEM_OUTPUT_TEXTURENB
+  texturenb:  Uses nearest-neighbour interpolation (no-bilinear).
```

Comment:
Minor: just "no bilinear" (drop the `-` please).

Reply:
Did a find/replace, which explains the commit to vga.h.